packages/grafana-e2e/cypress/plugins/index.js

```diff
@@ -1,6 +1,13 @@
 const fs = require('fs');
```

Comment:
This whole file is redundant to `packages/grafana-e2e/cypress.config.js` for grafana core but necessary to keep compatibility with Cypress 9 in plugins.